components/Layout/HomeSection.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <client-only>
     <swiper-section
-      :title="section.name"
+      :title="
+        section.libraryName === undefined
```

Comment:
Don't we need $t here for translating?

Reply:
That's the two lines below ;)

Reply:
@ThibaultNocchi *Late night* reviewing tales 🥱